proto/meta.proto

```diff
@@ -291,3 +291,13 @@ service ScaleService {
   rpc GetClusterInfo(GetClusterInfoRequest) returns (GetClusterInfoResponse);
   rpc Reschedule(RescheduleRequest) returns (RescheduleResponse);
 }
+
+message GetStateStoreUrlRequest {}
+
+message GetStateStoreUrlResponse {
+  string url = 1;
+}
+
+service StorageService {
+  rpc GetStateStoreUrl(GetStateStoreUrlRequest) returns (GetStateStoreUrlResponse);
```

Comment:
What about making this a return value in `AddWorkerNodeResponse` when registering a node to the meta service?